### DISCUSSION TITLE
Show injuries and afflictions post-mortem

### DISCUSSION THREAD

**Discussion by Veritius** [FakeFishGames/Barotrauma#5019] [Ideas and suggestions]
After a character has died, show the injuries and afflictions the character had when they died. This would be very useful and add a sort of deduction dynamic for finding traitors.

Additional idea, if a body is destroyed (as in the parts have seperated) do not show the afflictions.
Additional additional idea, if a body is shot with a flamer, the body is incinerated and no injuries or afflictions are shown apart from maximum burns.

**Comment by mLuby** [comment]
What about showing players a history of injuries when examining the health menu? The higher the character's medical skill, the older the injuries and afflictions they can discern.

**Comment by HeatedWafflez** [comment]
Admittedly I too miss this feature. Being able to perform autopsies was a crucial gameplay aspect that helped players to garner an idea of who might have committed a murder. For example, deducing that security was responsible for the death of a crew member because they're riddled with bullet hole, or in another instance deducing that it was medical because the victim was poisoned and only medical has access to poisons.
